src/plist.c

```diff
@@ -915,6 +915,9 @@ Obj    FuncEmptyPlist( Obj self, Obj len )
             "you can replace <len> via 'return <len>;'" );
     }
 
+    if(INT_INTOBJ(len) < 0) {
+        ErrorMayQuit("<len> must be greater than or equal 0",0L,0L);
```

Comment:
how about "must be non-negative" or  "must be a non-negative small integer" as we use elsewhere?